docs/sdks_02_cpk.md

```diff
@@ -14,4 +14,82 @@ The Contract Proxy Kit (CPK) seeks to bring the power of contract interactions t
 
 Currently, only batch transactions are directly supported by the CPK. This capability simplifies user flows for dapps that require multiple, sequential interactions with different contracts to be done in what should be a single transaction. For a prominent example, consider most dapps using the ERC20 standard. Often, dapps have to ask the user to approve an amount of ERC20 to be used before asking a dapp contract to pull the amount from the user’s account. This is a common pair of transactions which should be a single action, but would by default trigger two transactions when batch transactions are not available. Some more complex examples may involve coordinating interactions with multiple contract systems.
 
+### Adding the dependencies
 
+The contract proxy kit is only available as a js library via npm and can be added to your project with
+
+`npm install contract-proxy-kit`
+
+or
+
+`yarn add contract-proxy-kit`
+
+
+## Inititalizing the library
+
+The contract proxy kit can be used with Ethers or with Web3js. To initialize the contract proxy kit you need to provide these providers to the library.
```

Comment:
"Ethers" and "Web3js", both are js libraries. Should we use ("Ethersjs" and "Web3js") or ("Ethers" and "Web3")?

Reply:
called them ethers.js and web3.js (as that are the names they have on github :) )